### DISCUSSION TITLE
Loud distorted sound when using Streamlabs and Blackhole ch2... please help

### DISCUSSION THREAD

**Discussion by merumeru2006** [ExistentialAudio/BlackHole#789] [Q&A]
Hii,
I have blackhole ch2 installed
and I have Streamlabs
I want to record my microphone with my scarlet 2i2 and the audio from my Mac.

I can record the microphone audio with no problem
but when I try to record the audio coming from my Mac it does not work (the internal audio that comes from games or videos), there is just an extremely loud distorted buzzing sound and it records no audio from my Mac.

I have uploaded a video with my settings and the buzzing (BEWARE the sound is super loud in the video so turn down the volume if you are listening with headphones), or it records no sound at all 
https://youtu.be/MtEQ2b5O4t0

im rather confused, I tried following some YouTube tutorials and also looking online for people that might have had a similar problem but I didn't find anything

I tried following this tutorial: https://www.youtube.com/watch?v=A56Vb6eWYGQ
and also this https://streamlabs.com/content-hub/post/how-to-screen-record-on-mac
and I also tried this one https://www.youtube.com/watch?v=sT7YKMTdnX4

and I have this Mac: 
Apple MacBook Pro – Late 2021
16", M1 Max, 64 GB, 2000 GB
Memory | 64 GB
SSD capacity | 2000 GB
Operating System Version | Mac OS 12.0 (Monterey)
Graphics card model	
Apple M1 Max 32 Core
Processor type | M1 Max
Number of processor coresi | 10 -Core

(Im not sure which information is key for solving this problem, but I hope this gives more context)

I also tried turning my Mac on and off... it didn't fix the issue :-( 

I used to have a windows laptop but switched to Mac, therefore i don't have all the Mac knowledge...

Im super thankful for anyone that can help!!

**Comment by devinroth** [comment]
Sounds like you have a feedback loop going on.

I’m not an OBS expert so I can’t give you any specifics. Just make sure you aren’t monitoring the input or something like that.

Devin
